src/main/tray.ts

```diff
@@ -51,8 +52,30 @@ export class Tray {
 
   protected kubernetesState = State.STOPPED;
 
+  private isMacOs = () => {
+    return os.platform() === 'darwin';
+  }
+
+  private trayIconsMacOs = {
```

Comment:
I assume this should be `readonly` as well?

Reply:
I agree - updated to include the `readonly` modifier